app/services/alma_availability_service.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require 'rest-client'
+require 'nokogiri'
+
+class AlmaAvailabilityService
+  def initialize(mms_id)
+    @mms_id = mms_id
+    @response = query_availability
+    @xml = Nokogiri::XML(@response)
+  end
+
+  def current_availability
+    bib = {}
+    physical = @xml.at_xpath('bib/record/datafield[@tag="AVA"]/subfield[@code="e"]')
+    digital = @xml.at_xpath('bib/record/datafield[@tag="AVD"]')
+    electronic = @xml.at_xpath('bib/record/datafield[@tag="AVE"]')
+    bib[@mms_id] =
+      { physical: {
+        exists: physical ? true : false,
+        available: physical && physical.text == 'available' ? true : false
+      },
+        online: { exists: digital || electronic ? true : false } }
+    bib
+  end
+
+  def query_availability
+    RestClient.get api_url + "/almaws/v1/bibs/" + @mms_id + query_inst + api_key
```

Comment:
consider using string interpolation, as a matter of idiomatic style (either "foo/#{var}") or sprintf style ("%s"%[var])

might investigate if urlencoding query values is necessary (it might not be, but good practice in case)

Reply:
I'll switch to string interpolation. Commit upcoming.

Reply:
Commit pushed.